x-pack/test_serverless/functional/test_suites/observability/cases/attachment_framework.ts

```diff
@@ -32,18 +31,15 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         );
 
         await svlObltNavigation.navigateToLandingPage();
-
         await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'dashboards' });
 
         await dashboard.clickNewDashboard();
-
         await lens.createAndAddLensFromDashboard({});
-
         await dashboard.waitForRenderComplete();
       });
 
       after(async () => {
-        await cases.api.deleteAllCases();
+        await svlCases.api.deleteAllCaseItems();
```

Comment:
Needed to avoid permission errors when deleting directly a document from the `.kibana_alerting_cases` system index.